repo.go

```diff
@@ -110,6 +111,24 @@ func parseRepo(defaultSchema, defaultHost, defaultOrganization, name string) (re
 	return r, nil
 }
 
+func (r repo) isGithubRemote() (ok bool) {
+	cmd := xexec.Fmt("git remote -v")
```

Comment:
Shouldn't be using `xexec.Fmt` here since you're not using a format string.

Reply:
This won't work unless the current working directory is inside the repository, which isn't guaranteed. You can tell git to use a different path by giving it the `-C` parameter. Maybe `xexec.Fmt` is a good idea after all.